packages/dts-plugin/src/core/lib/DTSManager.ts

```diff
@@ -201,10 +201,20 @@ class DTSManager {
         return u;
       };
 
-      let publicPath =
-        'publicPath' in manifestJson.metaData
-          ? manifestJson.metaData.publicPath
-          : new Function(manifestJson.metaData.getPublicPath)();
+      let publicPath;
+
+      if ('publicPath' in manifestJson.metaData) {
+        publicPath = manifestJson.metaData.publicPath;
+      } else {
+        const getPublicPath = new Function(manifestJson.metaData.getPublicPath);
+
+        if (manifestJson.metaData.getPublicPath.startsWith('function')) {
+          publicPath = getPublicPath()(manifestJson.metaData);
```

Comment:
Why is the manifest json data passed in here?

Reply:
If we pass data into a callback, we will not be able to adjust once the user relies on the corresponding data

Reply:
it need to remove manifestJson.metaData arg as well